### PR TITLE
[TW-1451] Doc editing fork labels

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Changed gerunds to imperatives in headings and added a small section describing how to edit a forked element's name and label.

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/using-version-control/forking-entities.md [hunk 1]
```diff
@@ -140,4 +141,12 @@ To resolve a merge conflict from a fork, do the following:
 
 > To keep all changes on your fork, select **Keep all changes to source**. To keep all changes on the parent element, select **Keep all changes to destination**.
 
-After you resolve the conflicts, select **Pull changes** to [pull the updates](#pulling-updates-from-a-parent-element).
+After you resolve the conflicts, select **Pull changes** to [pull the updates](#pull-updates-from-a-parent-element).
+
+## Edit a fork
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/using-version-control/forking-entities.md:146]: I think moving this section up under Create a fork or View fork information might flow a little better. Could also consider renaming "Edit a fork". Personally it makes me think the section is about editing the actual forked collection/etc, vs the name/label. Maybe "Edit a fork's name and label", "Edit a fork's information"(?), etc would be more specific?

Alternatively could just add a brief sentence saying you can edit the fork as you would normally edit a collection/etc, and then also include the name/label piece

**Review by claireriley — APPROVED** [review]
LGTM! One minor note

File: src/pages/docs/collaborating-in-postman/using-version-control/forking-entities.md (hunk 1 above)
Comment (jkonrath-postman) [reply src/pages/docs/collaborating-in-postman/using-version-control/forking-entities.md:146]: I'd also say maybe "Edit a fork's name or label"

**Review by jkonrath-postman — APPROVED** [review]
LGTM, pending Claire's comment

File: src/pages/docs/collaborating-in-postman/using-version-control/forking-entities.md (hunk 1 above)
Comment (petewitcher-postman) [reply src/pages/docs/collaborating-in-postman/using-version-control/forking-entities.md]: Great suggestions, thanks guys.

**Pull request merged**